### ISSUE TITLE
Picker does not read out suggestions

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1610]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

When you type, and suggestions appear, we should tell them there are suggestions available. Arrowing down can read the suggestions.

**Comment by betrue-final-final** [comment]
In Narrator, doesn't announce when suggestions appear, and the tooltip says "People picker" in the example.

**Comment by kristoferbrown** [comment]
@betrue-final-final, PR #2493 improved the screen reader experience for pickers in a couple ways. This issue should be resolved now.
